src/components/DataListSection.tsx

```diff
@@ -77,36 +77,49 @@ export const DataListSection = ({
     skipLastDivider: true
   });
 
+  if (listData?.length) {
+    return (
+      <View>
+        {!!sectionTitle && (
+          <SectionHeader onPress={navigate} title={sectionTitle ?? getTitleForQuery(query)} />
+        )}
+        {!!limit &&
+          (listData?.length ? (
+            <ListComponent
+              data={isRandom ? _shuffle(listData).slice(0, limit) : listData.slice(0, limit)}
+              horizontal={horizontal}
+              navigation={navigation}
+              query={query}
+            />
+          ) : (
+            !!placeholder && placeholder
+          ))}
```

Comment:
this `placeholder` part can be removed now because we always have list data for this whole render part here, right?

Reply:
https://github.com/ikuseiGmbH/smart-village-app-app/pull/583/commits/c572a17d9f6c37319dec493afd2b8a6a9c43d631